src/main/java/de/tum/in/www1/artemis/versioning/VersionRangeService.java

```diff
@@ -0,0 +1,25 @@
+package de.tum.in.www1.artemis.versioning;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Provides functionality to work with {link VersionRange} annotations.
+ */
+public class VersionRangeService {
+
+    /**
+     * Converts the values of the range in a list of integers
+     * @param range The range to convert
+     * @return The list of integers representing versions
+     */
+    public static List<Integer> versionRangeToIntegerList(VersionRange range) {
+        List<Integer> list = new ArrayList<>();
+        int[] versions = range.value();
+        for (int version : versions) {
+            list.add(version);
+        }
+        return list;
```

Comment:
```suggestion
       return List.of(range.value());
```
Would this also work?

Reply:
Sadly no. That's always the thing with int arrays.
But you can mark a stream boxed, then it unpacks the array. Pretty weird.

Reply:
What about `Arrays.asList(range.value())` with `java.util.Arrays`? Does that work?

Reply:
Don't think this works either but I added another solution.